packages/cli/internal/pkg/environment/environment.go

```diff
@@ -1,78 +1,72 @@
 package environment
 
 import (
+	"fmt"
 	"os"
+	"strings"
 
 	"github.com/aws/amazon-genomics-cli/internal/pkg/aws/ecr"
+	"github.com/aws/amazon-genomics-cli/internal/pkg/constants"
 )
 
+var AllEngines = []string{
```

Comment:
I really like that you added this list. I agree that now, it makes sense to move engine related code to a separate package. It doesn't have to be this PR though. We can do that later. Another thing that we may want to do, is a quick unit test, that makes sure, that AllEngine list is in sync with all the maps. To catch cases when we added something to AllEngines, but forget to add it to UsesWesAdapter or another way around.